lib/smart_proxy_dynflow_core/log.rb

```diff
@@ -67,20 +112,32 @@ def call(_severity, _datetime, _prog_name, message)
           message
         end
       end
-
-      def format(message)
-        call(nil, nil, nil, message)
-      end
     end
 
     class ProxyAdapter < ::Dynflow::LoggerAdapters::Simple
       def initialize(logger, level = Logger::DEBUG, _formatters = [])
         @logger           = logger
         @logger.level     = level
-        @logger.formatter = ProxyStructuredFormater.new(@logger)
         @action_logger    = apply_formatters(ProgNameWrapper.new(@logger, ' action'), [ProxyStructuredFormater])
         @dynflow_logger   = apply_formatters(ProgNameWrapper.new(@logger, 'dynflow'), [ProxyStructuredFormater])
       end
     end
   end
 end
+
+class PryAction < ::Dynflow::Action
+  require 'pry'
+
+  def plan(args)
```

Comment:
This method should not be needed: the default one just expects the `args` to be hash and passes it as action input